java-support/tck/src/main/java/io/cloudstate/javasupport/tck/model/action/ActionTckModelBehavior.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2019 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cloudstate.javasupport.tck.model.action;
+
+import akka.NotUsed;
+import akka.actor.ActorSystem;
+import akka.stream.javadsl.Flow;
+import akka.stream.javadsl.Sink;
+import akka.stream.javadsl.Source;
+import io.cloudstate.javasupport.Context;
+import io.cloudstate.javasupport.ServiceCall;
+import io.cloudstate.javasupport.action.*;
+import io.cloudstate.javasupport.action.Effect;
+import io.cloudstate.tck.model.Action.*;
+import java.util.concurrent.CompletionStage;
+
+@Action
+public class ActionTckModelBehavior {
+
+  private final ActorSystem system = ActorSystem.create("ActionTckModel");
+
+  public ActionTckModelBehavior() {}
+
+  @CallHandler
```

Comment:
Just a non-PR comment. Why did we choose CallHandler instead of CommandHandler? I don't understand why we need to change the semantics since the behavior is implicit in the type of Entity used, we shouldn't have to call it CallHandler just because it is a stateless entity. This is definitely the case with "pattern is having no pattern".

Reply:
For Actions or Functions (which are not Entities) I expect `EventHandler` may be the most commonly used in other similar solutions, but I think that was seen as confusing with event-sourced entities having a more specific use of event handling. In some discussions, just using `Handler` for actions was discussed. I'll let @jroper comment on why its `CallHandler` currently. Sticking to `CommandHandler` for both entities and actions seems okay to me too.

Reply:
@sleipnir, do you want to create an issue for the `CallHandler` naming? So we track discussion there and remember to make changes.